hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/HoodieCompactor.java

```diff
@@ -141,7 +148,7 @@ public List<WriteStatus> compact(HoodieCompactionHandler compactionHandler,
                                    String instantTime,
                                    String maxInstantTime,
                                    TaskContextSupplier taskContextSupplier) throws IOException {
-    return compact(compactionHandler, metaClient, config, operation, instantTime, maxInstantTime,
+    return compact(compactionHandler, metaClient, config, operation, instantTime, maxInstantTime, Option.empty(),
```

Comment:
Where is this codepath called? 

Should be also pass the correct instantRange here to make the codeflow consistent? The above code to get the correct instantRange can be moved to the compact function or into a helper.

Reply:
It is only used by the Flink compactor, which is DT compaction so the instant range is not necessary, I will add some comments here.